.emacs.d/config.org

```diff
@@ -442,6 +443,11 @@ brew cask install font-fira-mono
     )
 #+END_SRC
 
+** Ag - The Silver Searcher
+If you want to use Projectile's search functions with ag
+
+(use-package ag)
```

Comment:
I'm still an ack man, but this might be the introduction I need.

Reply:
My bad. I originally made it just for myself, then later added Nate. I've added everyone else as followers since we're the only ones that use Emacs right now.